lib/server.js

```diff
@@ -107,7 +107,7 @@ function Server (torrent, opts) {
         return serveIndexPage()
       }
 
-      var index = Number(pathname.slice(1))
+      var index = Number(pathname.split('/')[0])
```

Comment:
Wow, so it happened to be that I was always testing with `0` as the file index. `Number('')` resolves to 0. Will update the PR